docs/layouts/partials/hooks/body-end.html

```diff
@@ -5,7 +5,7 @@
     container: '#docsearch',
     appId: 'ED67NSVSZS',
     apiKey: '21e2920c3a9c29b98cbc382694413f29',
-    indexName: 'radapp-dev-edge',
+    indexName: 'radapp-dev',
```

Comment:
i believe this should remain as `edge` if you're targeting the `edge` branch

Reply:
Updated